### PR TITLE
AP_Bootloader: reserve board id for Aocoda-RC & add board id for Aocoda-RC H743Dual

### PR DESCRIPTION
**Pull request by aocodarc**
AP_Bootloader: reserve board id for Aocoda-RC & add board id for Aocoda-RC H743Dual

### CODE REVIEW

File: Tools/AP_Bootloader/board_types.txt [hunk 1]
```diff
@@ -266,11 +266,15 @@ AP_HW_VIMDRONES_MOSAIC_X5            1405
...
 # IDs 5000-5099 reserved for Carbonix
+# IDs 5100-5199 reserved for SYPAQ Systems
 # IDs 5200-5209 reserved for Airvolute
-AP_HW_AIRVOLUTE_DCS2             5200
+AP_HW_AIRVOLUTE_DCS2                 5200
+
+# IDs 5210-5219 reserved for Aocoda-RC
+AP_HW_AOCODA-RC-H743DUAL             5210
 
-# IDs 5100-5199 reserved for SYPAQ Systems
```
Comment (peterbarker) [inline Tools/AP_Bootloader/board_types.txt:273]: Why are you moving this?

**Review by peterbarker — CHANGES_REQUESTED** [review]
Commit list needs fixing; perhaps just two commits, one fixing the airvolute format and one for the new reservation?

We don't allow merge commits in our commit history.

File: Tools/AP_Bootloader/board_types.txt (hunk 1 above)
Comment (aocodarc) [reply Tools/AP_Bootloader/board_types.txt:273]: Numbers（board id） are not very continuous.

**Comment by aocodarc** [comment]
> Commit list needs fixing; perhaps just two commits, one fixing the airvolute format and one for the new reservation?
> 
> We don't allow merge commits in our commit history.

ok

**Pull request closed without merging**